inference-engine/tests_deprecated/behavior/vpu/myriad_tests/vpu_watchdog_tests.cpp

```diff
@@ -182,20 +183,15 @@ TEST_P(MYRIADWatchdog, watchDogIntervalDefault) {
     auto startup_devices = queryDevices();
     auto ctime = Time::now();
     {
-
         InferenceEngine::Core core;
-        IE_SUPPRESS_DEPRECATED_START
-        CNNNetReader reader;
-        reader.ReadNetwork(GetParam().model_xml_str.data(), GetParam().model_xml_str.length());
-
-        CNNNetwork network = reader.getNetwork();
-        IE_SUPPRESS_DEPRECATED_END
+        auto model = FuncTestUtils::TestModel::convReluNormPoolFcModelFP16;
+        CNNNetwork network = core.ReadNetwork(model.model_xml_str, model.weights_blob);
```

Comment:
@nikita-kud could you please take a look. It's unclear to me - what's expected to check in this test

Reply:
This works locally:
```
ASSERT_NO_THROW(ret = core.LoadNetwork(network, GetParam().device, {
            {KEY_LOG_LEVEL, LOG_INFO},
            {KEY_VPU_MYRIAD_WATCHDOG, NO}}));
```

Reply:
thanks, fixed!